src/term.c

```diff
@@ -2300,7 +2300,7 @@ set_termname(char_u *term)
     full_screen = TRUE;		// we can use termcap codes from now on
     set_term_defaults();	// use current values as defaults
 #ifdef FEAT_TERMRESPONSE
-    LOG_TR("setting crv_status to STATUS_GET");
+    LOG_TR("setting crv_status to STATUS_GET","");
```

Comment:
Is this correction necessary even without changing L70's `#  if 0  // Change to 1 to enable ch_log() calls for termresponse debugging.` to `if 1`?
If possible, could you show me the compiler messages from before the correction?

Reply:
Ah, `##__VA_ARGS__` is a gcc extension (and clang and msvc are also fine). But the OpenVMS compiler doesn't support it...?